kymatio/scattering1d/tests/test_filters_scattering1d.py

```diff
@@ -1,16 +1,9 @@
 """
 Testing all functions in filters_bank
 """
-from kymatio.scattering1d.filter_bank import adaptive_choice_P
-from kymatio.scattering1d.filter_bank import periodize_filter_fourier
-from kymatio.scattering1d.filter_bank import get_normalizing_factor
-from kymatio.scattering1d.filter_bank import compute_sigma_psi
-from kymatio.scattering1d.filter_bank import compute_temporal_support
-from kymatio.scattering1d.filter_bank import compute_xi_max
-from kymatio.scattering1d.filter_bank import morlet_1d
-from kymatio.scattering1d.filter_bank import calibrate_scattering_filters
-from kymatio.scattering1d.filter_bank import get_max_dyadic_subsampling
-from kymatio.scattering1d.filter_bank import gauss_1d
+from kymatio.scattering1d.filter_bank import adaptive_choice_P, periodize_filter_fourier, get_normalizing_factor, \
```

Comment:
Parentheses around the imports so we don't have to use `\`.

Reply:
wow I cannot believe that you can do that.